fuel/app/classes/presenter/welcome/404.php

```diff
@@ -16,6 +16,25 @@ class Presenter_Welcome_404 extends Presenter
 	 */
 	public function view()
 	{
+		$fufufu = "jjkpowek;rlwke;lk;lk;lk";
+		$fufufddddu = "jjkpowek;rlwke;lk;lk;lk";
+
+	$fufufdddddddu = "jjkpowek;rlwke;lk;lk;lk";
```

Comment:
__[development][code_sniffer]__
* String "jjkpowek;rlwke;lk;lk;lk" does not require double quotes; use single quotes instead